src/pages/Encounters/components/EncounterListSkeleton.tsx

```diff
@@ -0,0 +1,37 @@
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+import { Separator } from "@/components/ui/separator";
+import { Skeleton } from "@/components/ui/skeleton";
+
+export default function EncounterListSkeleton() {
```

Comment:
This approach doesn't work, if we are moving skeltons to a single file it should be a generic file and should be part of the common components. 

Here is an example from Pupilfirst, https://github.com/pupilfirst/pupilfirst/blob/master/app/frontend/shared/components/SkeletonLoading.res